### PR TITLE
decomp appendGfx_player, fix warnings in 77480.c file

### PR DESCRIPTION
**Pull request by Rainchus**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.03%) on `us`.

✅ This PR fixes 47 warnings!

File: include/common_structs.h [hunk 1]
```diff
@@ -1765,7 +1765,8 @@ typedef struct PlayerStatus {
     /* 0x000 */ s32 flags;
     /* 0x004 */ u32 animFlags;
     /* 0x008 */ s16 framesOnGround; /* Number of frames since last jump landed */
-    /* 0x00A */ char unk_0A[2];
+    /* 0x00A */ s8 unk_0A;
+    /* 0x00A */ char unk_0B;
```
Comment (Brotenko) [inline include/common_structs.h:1769]:
```suggestion
    /* 0x00B */ char unk_0B;
```

File: src/77480.c [hunk 2]
```diff
@@ -750,7 +785,68 @@ void render_player(void) {
...
 INCLUDE_ASM(void, "77480", render_player_model);
 
-INCLUDE_ASM(s32, "77480", appendGfx_player);
+void appendGfx_player(void) {
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    Matrix4f sp20, sp60, spA0, spE0;
+    f32 temp_f0;
+    s32 phi_a0;
+
+    temp_f0 = -gCameras[gCurrentCamID].currentYaw;
```
Comment (Brotenko) [inline src/77480.c:794]: Can you declare during initialization or does it have to be after it?

**Review by Brotenko — CHANGES_REQUESTED** [review]

File: src/77480.c (hunk 2 above)
Comment (Rainchus) [reply src/77480.c]: declaring during initialization works

**Review by Brotenko — APPROVED** [review]

File: include/enums.h [hunk 3]
```diff
@@ -1411,8 +1411,8 @@ enum ActionStates {
     ACTION_STATE_ULTRA_POUND                    = 0x00000010,
     ACTION_STATE_SLIDING                        = 0x00000011,
     ACTION_STATE_HAMMER                         = 0x00000012,
-    ACTION_STATE_13                             = 0x00000011,
-    ACTION_STATE_14                             = 0x00000012,
```
Comment (nanaian) [inline include/enums.h:1415]: Lol??

**Review by nanaian — APPROVED** [review]

**Pull request merged**